java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/LongByteArrayMapTest.java

```diff
@@ -141,8 +152,18 @@ public void pipelinedGet(final ThreadState state, @StartNanos final long startNa
     public class ThreadState extends BaseThreadState {
         private Pipelining<byte[]> pipeline;
         private int i;
+        private int fixedKeysDomain;
```

Comment:
Since we already have `keyDomain` field, could we align the names, i.e. `fixedKeysDomain` -> `fixedKeyDomain`?

Also consider using the same convention for `fixedKeysPercentage` and `fixedKeysProbability` fields.